forem/forem
# Backfill community_emoji data with a DUS 🌱

## What type of PR is this? (check all applicable)

- [x] Preparation for removal ✂️ 

## Description

## Related Tickets & Documents
- Related Issue #15877 

## QA Instructions, Screenshots, Recordings
To QA this PR, please run the data update script (data_updates:run) and ensure that your community name updates accordingly, appending your community emoji to your community name. If your community does not have a community emoji, then the script should run without breaking or appending anything. 

### UI accessibility concerns?
N/A

## Added/updated tests?

      have not been included_

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Storybook](https://storybook.forem.com/) (for Crayons components)
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
      or in a [forem.dev](http://forem.dev) post
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] Are there any post deployment tasks we need to perform?
This DUS should run on all Forems.

## [optional] What gif best describes this PR or how it makes you feel?

![alt_text](gif_link)
